google_compute_engine/distro/utils.py

```diff
@@ -37,3 +37,13 @@ def EnableNetworkInterfaces(
       dhclient_script: string, the path to a dhclient script used by dhclient.
     """
     pass
+
+  def HandleClockSync(self, logger):
+    """Sync the software clock with the hypervisor clock.
```

Comment:
Can you update the other distro util files to have the same description? Also, please remove the "Called when..." line below.

Reply:
Done